engine/apps/api/tests/test_alert_receive_channel.py

```diff
@@ -1879,6 +1879,49 @@ def test_update_other_integration_additional_settings(
     assert response.status_code == status.HTTP_400_BAD_REQUEST
 
 
+@pytest.mark.django_db
+def test_create_integration_default_webhooks(
+    make_organization_and_user_with_plugin_token,
+    make_user_auth_headers,
+):
+    organization, user, token = make_organization_and_user_with_plugin_token()
+    integration = AlertReceiveChannel._config[0]
+    integration.additional_settings_serializer = AdditionalSettingsTestSerializer
```

Comment:
Note that I'm [refactoring this](https://github.com/grafana/oncall/pull/4050/files#diff-43488e01a34d0e4691f1aecb4765b5ea1b8f818c5eed09781bf342b7e15e0481R1771) because otherwise the config will be updated for every other test.